tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -645,11 +645,11 @@ def test_clock_cycles(dut):
 
     yield RisingEdge(clk)
 
-    dut.log.info("After one edge")
+    dut._log.info("After one edge")
 
     yield ClockCycles(clk, 10)
 
-    dut.log.info("After 10 edges")
+    dut._log.info("After 10 edges")
```

Comment:
How did this work before?

Reply:
Due to this: https://github.com/potentialventures/cocotb/blob/ed3e6d2dd4e870c40462d4385cc27779979bc961/cocotb/handle.py#L66